README.md

```diff
@@ -1,5 +1,6 @@
-# FSS organisation re-verification process
+This application and it's cloud recources has now been decommissioned
 
+<!--
```

Comment:
```suggestion
```
Remove comment block to reinstate text (matching the suggestion above). nit: If we're removing the text we should delete it rather than commenting it out.

Reply:
That was my bad. During last decommissioning I told them to comment out the README as the entire readme was merely a boiler plate on how to spin app up. This doesn't apply in this case.